src/lib/ngsi10/UpdateContextRequest.cpp

```diff
@@ -70,6 +75,28 @@ UpdateContextRequest::UpdateContextRequest(const std::string _contextProvider, E
 */
 void UpdateContextRequest::init(void)
 {
+  xmls  = 0;
+  jsons = 0;
+}
+
+
+
+/* ****************************************************************************
+*
+* UpdateContextRequest::format - 
+*/
+Format UpdateContextRequest::format(void)
+{
+  if (xmls > jsons)
```

Comment:
Wait, this is not correct. I means, right after receiving the response from mongoBackend, BEFORE sending the forwarded message (which is where the format is used :-))